pkg/worker/cedana.go

```diff
@@ -68,8 +70,8 @@ func NewCedanaClient(
 		fmt.Sprintf("--port=%d", port),
 		fmt.Sprintf("--gpu-enabled=%t", gpuEnabled))
 
-	daemon.Stdout = os.Stdout
-	daemon.Stderr = os.Stderr
+	daemon.Stdout = &common.ZerologIOWriter{LogFn: func() *zerolog.Event { return log.Info().Str("operation", "cedana daemon start") }}
+	daemon.Stderr = &common.ZerologIOWriter{LogFn: func() *zerolog.Event { return log.Error().Str("operation", "cedana daemon start") }}
```

Comment:
@luke-lombardi Here is how I'm capturing the output from exec's and structuring them. In the case of cedana, it would be nice if they added a flag to not use the pretty printing in zerolog. If they did that, we could remove this and we would also be able to query over fields they log from the daemon.